### PR TITLE
Debian VPS installation feedback

### PR DESCRIPTION
**Pull request by Djiko**
Hi all,

a couple of update proposals after a fresh installation on a Debian VPS. I run into 2 issues and did not immediately find the solution. This might help others.

Keep up the good work.

### CODE REVIEW

File: install_manually.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Once you have access to your server, either directly or by SSH, you can install
 
 1. Install git
 ```bash
-sudo apt-get install git
+sudo apt-get install git dialog
```
Comment (M5oul) [inline install_manually.md]: Why do you propose `dialog` package?
Can't we install it from install script or YunoHost debian dependency?
Installing `git` is only to clone `install_script` repository.

**Comment by M5oul** [comment]
The rest of the modification is ok for me.

**Comment by Djiko** [comment]
Dialog is used to display windows such as this one in the shell

https://yunohost.org/images/install_script.png

If it's not installed, you don't see anything, the installation process 
doesn't hang but doesn't process either as it is not able to display the 
needed windows. Debian VPS might not have it installed.

This is the way I solved it (easier for me, did not have to touch the 
install script at all), but as the install script already 
installs/updates some packages, this is certainly the best way to go.

File: install_manually.md (hunk 1 above)
Comment (M5oul) [inline install_manually.md]: Ok, I didn't found where to install `dialog` on the install script.
I am ok to install it here.

**Pull request merged**